fronts-client/src/constants/url.ts

```diff
@@ -14,14 +14,19 @@ export default {
 		capi: 'content.guardianapis.com',
 	},
 	media: {
-		apiBaseUrl: pageConfig.apiBaseUrl,
+		apiBaseUrl: pageConfig.baseUrls.apiBaseUrl,
 		mediaBaseUrl: 'https://media',
 		usageBaseUrl: '/api/usage',
 		imgIXDomainExpr: /^https:\/\/i\.guim\.co\.uk\/img\/static\//,
 		imageCdnDomain: '.guim.co.uk',
 		staticImageCdnDomain: 'https://static.guim.co.uk/',
 		imageCdnDomainExpr: /^https:\/\/(.*)\.guim\.co\.uk\//,
 	},
+	video: {
+		videoBaseUrl: 'https://video',
+		mediaAtomMakerPath: '/videos/',
+		capiAtomPath: 'atom/video/',
```

Comment:
Would `capiVideoAtomPath` be clearer to prevent confusion with interactive atoms?

Reply:
I also cant see this being used anywhere - is this used in a future PR?